lib/practice_exercises.rb

```diff
@@ -1,13 +1,45 @@
 
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n)
+# Space Complexity: O(1)
 def remove_duplicates(list)
-  raise NotImplementedError, "Not implemented yet"
+  index = 0
+  prev_num = 0
+  list.length.times do
+    item = list[index]
+    if index == 0
+      prev_num = item
+      index += 1
+    else
+      if item == prev_num
+        list.delete_at(index)
```

Comment:
`delete_at` actually shifts all the subsequent elements over 1 index.  So it actually has a runtime of O(n).  Since it's inside a loop running n times... Your method has a time complexity of O(n<sup>2</sup>)